libs/sdl/gl.c

```diff
@@ -381,7 +392,14 @@ HL_PRIM vdynamic *HL_NAME(gl_create_framebuffer)() {
 
 HL_PRIM void HL_NAME(gl_bind_framebuffer)( int target, vdynamic *f ) {
 	GLOG("%d,%d",target,ZIDX(f));
+#if	defined(HL_IOS) || defined(HL_TVOS)
+	SDL_SysWMinfo info;
+	SDL_VERSION(&info.version);
+	SDL_GetWindowWMInfo(SDL_GL_GetCurrentWindow(), &info);

```

Comment:
Is the value returned constant ? if yes, could this be done + stored in a static local var so the calls are not made for every framebuffer change ? (or at least if ZIDX(f) == 0 in the worst case)

Reply:
Fixed in https://github.com/HaxeFoundation/hashlink/pull/101/commits/1af597801d7ba76741dd36435f13b33acf9bbe16